### ISSUE TITLE
Vercel build times out...

### ISSUE THREAD

**Issue by michaelweinold** [diegofcornejo/vercel-jupyter-lite#3]
Hi, @diegofcornejo! Thank you for providing this template repo.

Having simply forked this project, it seems that the Vercel build times out at:

```
Dependencies resolved.
================================================================================
 Package          Arch        Version                    Repository        Size
================================================================================
Installing:
 wget             x86_64      1.21.3-1.amzn2023.0.3      amazonlinux      780 k
Installing dependencies:
 libmetalink      x86_64      0.1.3-14.amzn2023.0.2      amazonlinux       31 k
Transaction Summary
================================================================================
Install  2 Packages
Total download size: 811 k
Installed size: 3.3 M
```

Is it still working for you?

**Comment by diegofcornejo** [comment]
Hi @michaelweinold , thanks for the information. 

The issue seems to be related to the Node.js 20 version in the Vercel runtime. I did a quick test, and it indeed doesn't install the necessary dependencies.
In your project's settings on Vercel, please select Node.js 18 under Settings/General/Node.js Version, and everything should work fine. 
In the meantime, I'll investigate the installation issue with Node.js 20.

**Comment by diegofcornejo** [comment]
Hi @michaelweinold,

Thank you for pointing out the issue. 

I've identified that the Node.js 20 version running on Amazon Linux 2023 requires the libmetalink-0.1.3-14.amzn2023.0.2.x86_64 package as a dependency when installing wget. 

To address this, I've added the -y flag to the installation command, which automatically accepts all necessary prompts during the installation process. I've updated the script accordingly and committed the changes.

```sh
yum install wget -y
```
